experimental/pages/demo_home/views/demo_home_view.dart

```diff
@@ -1,21 +1,51 @@
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
+import '../../../routes/demo_app_pages.dart';
 import '../controllers/demo_home_controller.dart';
+import '../../../components/atoms/menu_list_tile.dart';
 
 class DemoHomeView extends GetView<DemoHomeController> {
   const DemoHomeView({super.key});
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Padding(
-        padding: const EdgeInsets.only(top: 50.0),
-        child: Align(
-          alignment: Alignment.topCenter,
-          child: Text(
-              'ユーザID:${controller.authService.uid.val}'), //TODO: ログインユーザーのIDを表示
-        ),
+      body: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(top: 50.0),
+            child: Align(
+              alignment: Alignment.topCenter,
+              child: Text(
+                'ユーザID:\n${controller.authService.uid.val}',
+                style: Theme.of(context).textTheme.headlineSmall,
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: 50.0, left: 10.0, bottom: 10.0),
+            child: Align(
+              alignment: Alignment.centerLeft,
+              child: Text(
+                'ユーザー情報',
+                style: Theme.of(context).textTheme.subtitle1,
+              ),
+            ),
+          ),
+          Expanded(
+            child: ListView(
+              padding: EdgeInsets.zero,
+              children: <Widget>[
+                MenuListTile(title: 'プロフィール情報登録', isTop: true, onTap: () => { Get.toNamed(DemoRoutes.demoProfileCreate) },),
```

Comment:
細かい指摘多くて申し訳ないですが、改行してフォーマット合わせるところまでお願いします🙇
下記一例ですが、MenuListTileのところは改行してほしいです。
```suggestion
                MenuListTile(
                    title: 'プロフィール情報登録', 
                    isTop: true,
                    onTap: () {
                       Get.toNamed(DemoRoutes.demoProfileCreate);
                     },
                ),
                ...
```

Reply:
@king-kazu39
ご指摘ありがとうございます！
修正しました！